.github/workflows/hackage.yml

```diff
@@ -78,16 +78,32 @@ jobs:
           cache-name: hackage-sources
         with:
           path: ${{ env.CABAL_PKGS_DIR }}
-          key:          ${{ env.cache-name }}-${{ env.INDEX_STATE }}
+          key:  ${{ env.cache-name }}-${{ env.INDEX_STATE }}
           restore-keys: ${{ env.cache-name }}-
 
+      # To ensure we get the lastest hackage index and not relying on haskell action logic
```

Comment:
Note that specifically so far was not including `hackage.yml` into the dev loop. As wanted to keep it as stable & untouched as possible. To first iron-out setup in PR CI loop & then spread the results in release workflow.

Reply:
I see, well, i think we have to make all uniform sooner or later.
I tried the workflow in my branch and it looks fine: https://github.com/jneira/haskell-language-server/runs/4695347296?check_suite_focus=true